conda/recipe/meta.yaml

```diff
@@ -64,12 +64,12 @@ requirements:
   host:
     - python
     - future >=0.17.1
-    - protobuf >=3.17.1
-    - libprotobuf-static >=3.17.1
+    - protobuf >=3.17.2
+    - libprotobuf-static >=3.17.2
     - libjpeg-turbo >=2.1.0
-    - tensorflow-gpu =2.2.0
-    - tensorflow-estimator =2.2.0
-    - tensorboard =2.2.2
+    - tensorflow-gpu =2.4.1
```

Comment:
do we force users to install TF?

Reply:
Yes. We do. We need to prebuild plugin and have a TF version defined. Maybe in future we can think how to compile in fly during the installation (if that is possible at all).